### PR TITLE
sixMans AutoMove

### PR DESCRIPTION
**Pull request by BenjaminHunt**
adds setting for sixMans queues to automatically move members to their team voice channel if they are connected to a vc. (Default: False)

- `<p>toggleAutoMove` enables or disables the bot from automatically moving members to their sixMans team voice channels
- Players are moved in `SixMans._randomize_teams` if `AutoMove` is set to `True`.

### CODE REVIEW

File: sixMans/sixMans.py [hunk 1]
```diff
@@ -947,8 +997,12 @@ async def _create_game(self, ctx, six_mans_queue):
         for channel in six_mans_queue.channels:
             await channel.send("**Queue is full! Game is being created.**")
         text_channel, voice_channels = await self._create_game_channels(ctx, six_mans_queue)
+
+
```
Comment (adammast) [inline sixMans/sixMans.py:1001]: I know this is nitpicky, but was there a reason these spaces got added? I can understand 1, but not sure why there's 3 here.

File: sixMans/sixMans.py [hunk 2]
```diff
@@ -221,6 +221,29 @@ async def q(self, ctx):
...
+        if player in six_mans_queue.queue.queue:
+            await ctx.send(":x: You are already in the {0} queue".format(six_mans_queue.name))
+            return
+        for game in self.games:
+            if player in game:
+                await ctx.send(":x: You are already in a game")
+                return
+
+        automove = await self._get_automove(ctx)
```
Comment (adammast) [inline sixMans/sixMans.py:242]: This variable doesn't look like it's used here at all and we end up getting it again in the randomize teams method so I assume this is just left over from testing

File: sixMans/sixMans.py [hunk 3]
```diff
@@ -221,6 +221,29 @@ async def q(self, ctx):
         if six_mans_queue._queue_full():
             await self._randomize_teams(ctx, six_mans_queue)
 
+    @commands.guild_only()
+    @commands.command(aliases=["forcequeue"])
+    @checks.admin_or_permissions(manage_guild=True)
+    async def fq(self, ctx, *, member: discord.Member):
```
Comment (adammast) [inline sixMans/sixMans.py:227]: I'm not sure if you noticed the queueAll command above, but it seems like this new command tests the same functionality that the queueAll command is there to test so I don't think this is necessary unless there's something new I'm missing.

File: sixMans/sixMans.py (hunk 1 above)
Comment (BenjaminHunt) [reply sixMans/sixMans.py:1001]: Nope. No reason at all. Must have been a mistake. I'll have them removed.

File: sixMans/sixMans.py (hunk 2 above)
Comment (BenjaminHunt) [reply sixMans/sixMans.py:242]: This was leftover code from when I was deciding where the behavior should exist. It is unused and should be removed.

File: sixMans/sixMans.py (hunk 3 above)
Comment (BenjaminHunt) [reply sixMans/sixMans.py:227]: I did see it. However, I didn't want members of my test server getting pinged for my tests. So I manually queued specific bots or people who didn't mind.

Happy to have it removed, but it doesn't make a huge difference.

File: sixMans/sixMans.py (hunk 2 above)
Comment (BenjaminHunt) [reply sixMans/sixMans.py]: Done

File: sixMans/sixMans.py (hunk 1 above)
Comment (BenjaminHunt) [reply sixMans/sixMans.py]: Done

**Comment by BenjaminHunt** [comment]
I moved the automove functionality inside the `Game.add_to_blue` and `Game.add_to_orange` functions.

File: sixMans/sixMans.py (hunk 3 above)
Comment (adammast) [reply sixMans/sixMans.py:227]: The queueAll command takes in a list as a parameter so you can specify the list of members you want added. I'm not sure if there was confusion on that or not. The only difference I see between this command and the queueAll command is that queueAll takes in at least one member and potentially multiple whereas this one takes in only one at a time. I think it's redundant in that queueAll can be used in the same manner if desired.

Comment (adammast) [reply sixMans/sixMans.py:227]: Unless there's some functionality here that I'm missing I think this should be removed and then an alias for queueAll can be added to work with the same command name.

Comment (BenjaminHunt) [reply sixMans/sixMans.py:227]: In that case, there's no difference and it should be removed. I misunderstood the behavior of queueAll. I'll have the forcequeue command removed.

**Review by adammast — APPROVED** [review]

**Pull request merged**